src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -472,6 +472,16 @@ limitations under the License.
       <arg name="arg1" type="int8u"/>
     </command>
 
+    <command source="client" code="0x18" name="StringEchoRequest"
+             response="StringEchoResponse" optional="true">
+      <description>
+        Command that takes an argument which is an octet string.  The response echoes
+        the string back. If the string is large then it would require a session that
```

Comment:
the command code should be consecutive, move this definition after TestSecondBatchHelperRequest (code="0x17")

Reply:
Done.